hhnk_research_tools/logger.py

```diff
@@ -0,0 +1,130 @@
+# %%
+"""
+These functions allow for adding logging to the console.
+This is applied by default in the __init__ of hhnk_research_tools. So when it is imported
+in a project, the logging will be set according to these rules.
+"""
+
+import logging
+from logging import *  # noqa: F401,F403 # type: ignore
+
+
+def get_logconfig_dict(level_root="WARNING", level_dict={}, log_filepath=None):
```

Comment:
beter om geen dictionaries zo als default voor parameters te gebruiken, zie https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python
None werkt hier prima

Reply:
Ah, ja daar was ik al half aan begonnen om daar None van te maken maar dus nog niet afgemaakt.